scripts/features/solr.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Install Java Runtime Enviroment
+sudo apt-get update
```

Comment:
@svpernova09 can correct me if I'm wrong, but I believe we decided on switching to `apt` instead of `apt-get` for v9.

Reply:
I'm somewhat torn, while I would like to switch to apt, and we currently have in V9, even apt gives CLI warnings that the CLI interface isn't stable, which has me rethinking it.

Reply:
interesting answer here. https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning

maybe we were wrong on this.  sounds like we should use things like `apt-get` and `apt-cache` for scripts, but users running straight from the command line should use `apt`.